tools/cellprofiler/cellprofiler.xml

```diff
@@ -0,0 +1,75 @@
+<tool id="cp319" name="Cellprofiler" version="19.05">
+  <description>Run a CellProfiler v3.1.9 pipeline</description>
+  <requirements>
+     <requirement type="package" version="3.1.9">cellprofiler</requirement>
+  </requirements>
+  <configfiles>
+<configfile name="script_file">
+mkdir `pwd`/input
+mkdir `pwd`/output
+#for $i,$filename in enumerate($inputType.inputdir):
+ln -s $filename `pwd`/input/${filename.element_identifier}.${filename.ext}
+#end for
+
+find `pwd`/input -name *.tiff > list.txt
+
+</configfile>
+  </configfiles>
+
+  <command><![CDATA[
+      bash '$script_file' &&
+      cellprofiler -c -r --file-list list.txt -o `pwd`/output -p $pipeline 
+  ]]></command>
+  <inputs>
+	  <param name="pipeline" type="data" format="txt" label="Pipeline file"/>
+    
+    <conditional name="inputType">
+       <param name="iType" type="select" label="Select either a zip file or a collection from history">
+          <option value="c">collection</option>
+          <option value="z">zip file</option>
```

Comment:
* is zip really needed?
* collection should be removed and instead the data-param should be a 'multiple="true"'

Reply:
-zip maybe useful, i just leave it here for now, can be removed anytime
- changed to multiple true